src/redis-cli.c

```diff
@@ -46,8 +46,8 @@
 #include <limits.h>
 #include <math.h>
 
-#include "hiredis.h"
-#include "sds.h"
+#include <hiredis.h>
+#include <sds.h> /* use sds.h from hiredis, so that only one set of sds functions will be present in the binary */
```

Comment:
Why not making redis-cli just use the (new) Redis versions of sds.[ch]? Because it exchanges SDS strings with hiredis? Thanks.